### ISSUE TITLE
Gene Summary - History widget

### ISSUE THREAD

**Issue by tuli** [WormBase/website#3812]
If you look at e.g. http://www.wormbase.org/species/c_elegans/gene/WBGene00017132 you will see the History widget lists Actions and Remarks.

When there is more than one Action for a single date please can you put them on individual lines. It will then follow that the associated Remark will be listed alongside the Action. As it stands the Actions appear on a single line and not all the Remarks are displayed. 

e.g. 5	2014-9-02	CGC_name,Other_name	tofu-6		Mary Ann Tuli

Should be
5 2014-9-02	CGC_name	mel-47		Mary Ann Tuli
5 2014-9-02	Other_name	tofu-6		Mary Ann Tuli

**Comment by dasmoth** [comment]
Something that's puzzling me slightly and may or may not be related: in WS248, this gene has a Version of 4, but Version_changes going up to 5 -- any idea what's going on?

**Comment by tuli** [comment]
@dasmoth good catch. It's a mistake and has been fixed for WS249. I don't think it's related to the original issue though.
